framework/src/main/java/org/tron/common/application/ApplicationImpl.java

```diff
@@ -31,6 +32,9 @@
   @Autowired
   private ConsensusService consensusService;
 
+  @Autowired
+  private BlockChainMetricManager blockChainMetricManager;
```

Comment:
The setting is reversed. It should be metricsservice.init (blockchainmetricmanager. Setmetricsservice (this)))